tests/functional/m/missing/missing_timeout.py

```diff
@@ -0,0 +1,92 @@
+# pylint: disable=consider-using-with,import-error,missing-module-docstring,no-member,no-name-in-module,reimported
+
+import requests
+import serial
```

Comment:
I would be okay with only checks for ``requests``. @Pierre-Sassoulas what do you think?

Reply:
Do you mean in the example of in the checker itself ? (Please do not resolve discussion where the discussion did not happen please @moylop260 I missed that one, as I have hundred of mails per day I rely on the web interface :) )

Reply:
I have already removed since that I agree with Daniel about it was changed in the code too

Since it could be not the same for external request than internal port

Even, we can add it in the configuration than the documentation says

So, IMHO nothing to discuss here that is reason it is resolved